_sass/components/_icon-list.scss

```diff
@@ -0,0 +1,5 @@
+ul.usa-icon-list {
+    .page-content__prose & {
+        padding-left: 0;
```

Comment:
I was hoping to talk to you about this. The more I consider (thoughts below) the more I think I want to go back and fix this padding issue in the design system.
- I'm not 100%, that's not the only place I see this style fix in this site:
    - Searching `padding-left: 0` in this repo (`identity-site`) shows me 3 places in `_sass/components` where we set padding-left to 0 for a ul.
    - That starts to sound like a medium amount of work, to fix each of those in design system and here.

Reply:
Can you explain what the issue is? From what I recall, it was not intentional in the designs to diverge from the USWDS, and so the default styles that we're seeing take effect should be expected?

Reply:
**The short version of what I think is happening:**
- In the design sytsem repo we have no `padding-left` for the icon list.
- In the help center repo there is an existing style (I'm not sure where) that sets `padding-left: 3ch` for `ul` (and others).
- For this particular, `icon-list`, `ul` we don't want that padding.


**The longer issue description:**
- In the design system repo (and on the [USWDS page](https://designsystem.digital.gov/components/icon-list/)) the icons in the icon list have no left padding
![Screenshot 2023-08-31 at 09 50 50](https://github.com/18F/identity-site/assets/6818839/d193b14d-96c6-49d1-8986-e0fa9b2c3d47)
- In the help site repo using the same classes, produces an icon list with left padding. This is not desired, we'd like to have no padding (per [designs](https://www.figma.com/file/amLP3Kk1PFRnm6Bk82PbxV/LG-10238%3A-Visuals-of-do's-and-don'ts-for-image-capture?type=design&node-id=199-942&mode=design&t=TE7wx7IKJtV6TZqP-0))
![Screenshot 2023-08-31 at 09 54 11](https://github.com/18F/identity-site/assets/6818839/b0f1595f-87d6-4aff-a8b7-ef722293df81)
- That padding seems to be coming from `.page-content__prose ul`
![Screenshot 2023-08-31 at 09 46 12](https://github.com/18F/identity-site/assets/6818839/44d33018-29e4-42e1-8f2f-6f6ad76c7063)
- I'm assuming that for other `ul` (etc) in the help site we do desire this padding, so overriding/removing it fully isn't a good idea. At the very least it would be a much bigger feeling change.
- I found another instance where we override this pattern in the help site: `_sass/components/_list.scss` So my approach here is to basically re-use that override for the icon-list component.



Note: I'm adding black borders for easier visualization, they're not part of the issue.

Reply:
Ah, thanks for explaining. I think this is another case where this site's custom styles are interfering. We / I should probably address that separately as technical debt, but in the meantime, I think it's reasonable to have these styles to re-reset the padding.

Reply:
Thanks for taking a look. I'll proceed with this override for this PR.